rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -267,18 +270,20 @@ pub struct Neuron {
     ///
     /// Cf. \[Neuron::stop_dissolving\] and \[Neuron::start_dissolving\].
     #[prost(oneof = "neuron::DissolveState", tags = "9, 10")]
-    pub dissolve_state: ::core::option::Option<neuron::DissolveState>,
+    pub dissolve_state: Option<neuron::DissolveState>,
 }
 /// Nested message and enum types in `Neuron`.
 pub mod neuron {
+    use super::*;
+
     /// Protobuf representing a list of followees of a neuron for a
     /// specific topic.
     #[derive(candid::CandidType, candid::Deserialize, serde::Serialize, comparable::Comparable)]
     #[allow(clippy::derive_partial_eq_without_eq)]
     #[derive(Clone, PartialEq, ::prost::Message)]
     pub struct Followees {
         #[prost(message, repeated, tag = "1")]
-        pub followees: ::prost::alloc::vec::Vec<::ic_nns_common::pb::v1::NeuronId>,
```

Comment:
the prost::alloc::vec:: may be important - we still have to support proto "Governance" in the init method.  We want to phase this out soon, but it's still a requirement.

Reply:
Maybe it's not a good idea to change it, but `prost::alloc::vec::Vec` is the same type as `Vec`. This is why you can pass a `prost::alloc::vec::Vec<_>` to a function that expects `Vec<_>`. In fact the entire `prost::alloc` is the same as `std::alloc`.